frontend/src/pages/Landing.tsx

```diff
@@ -1,26 +1,24 @@
-import { Stack } from 'components/common';
-import { PackageCard } from 'components/cards';
-import { Greeting } from 'components/text';
+import { Greeting, PackageCard, Stack } from 'components';
+import { PackageType } from 'constants/packages';
+import { Link, useLoaderData } from 'react-router-dom';
 
 const Landing = () => {
+  const packages = useLoaderData() as PackageType[];
+
   return (
     <>
       <Greeting />
       <Stack gap="20px">
-        <PackageCard
-          icon="Authenticate"
-          title="SSO React"
-          pageRoute="/sso-react"
-          summary="An npm package that offers an integration solution for React applications requiring
-                authentication through the B.C. government's Single Sign-On SSO service."
-        />
-        <PackageCard
-          icon="Authenticate"
-          title="SSO Express"
-          pageRoute="/sso-express"
-          summary="An npm package that offers an integration solution for Express applications requiring
-                authentication through the B.C. government's Single Sign-On SSO service."
-        />
+        {packages.map((pkg) => (
+          <Link key={pkg.title} to={pkg.pageRoute} style={{ textDecoration: 'none' }}>
```

Comment:
The `PackageCard` component already has an onClick action. We should remove either the `pageRoute` prop for the `PackageCard` or the `Link` component.

Reply:
kept the Link component as it allows react router to deal with page changing within the dom